openpectus/aggregator/main.py

```diff
@@ -71,7 +70,7 @@ def get_unit(id: int) -> ProcessUnit:
     return ProcessUnit(
         id=3,
         name="Foo",
-        state=ProcessUnitState.Ready(),
+        state=ProcessUnitState.Ready(),  # type: ignore
```

Comment:
Does that comment mean that you need to explicitly ignore the type any time a ProcessUnitStateEnum value is used? Or is it a comment about something else?

Reply:
The type checker doesn't seem to completely understand the Literal types and gives an 'Argument missing for parameter "state"' for the ProcessUnitState.Ready() constructor. So it's slightly annoying, but luckily its only related to the constructor - not the type itself.